CreatubblesAPIClient/Sources/Requests/CustomStyle/CustomStyleFetchResponseHandler.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  CustomStyleFetchResponseHandler.swift
+//  CreatubblesAPIClient
+//
+//  Created by Michal Miedlarz on 03.06.2016.
+//  Copyright © 2016 Nomtek. All rights reserved.
+//
+
+import UIKit
+import ObjectMapper
+
+class CustomStyleFetchResponseHandler: ResponseHandler
+{
+    private let completion: CustomStyleClosure?
+    
+    init(completion: CustomStyleClosure?)
+    {
+        self.completion = completion
+    }
+    
+    override func handleResponse(response: Dictionary<String, AnyObject>?, error: ErrorType?)
+    {
+        if  let response = response,
+            let mapper = Mapper<CustomStyleMapper>().map(response["data"])
+        {
+            let metadataMapper = Mapper<MetadataMapper>().map(response["meta"])
```

Comment:
metadata and dataMapper should be extracted soon due to increased number of repetitions